src/test/java/ja/worshop/hibernate/CollectionsTest.java

```diff
@@ -0,0 +1,48 @@
+package ja.worshop.hibernate;
+
+import ja.workshop.hibernate.model.Author;
+import ja.workshop.hibernate.model.Book;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+/**
+ * @author Agnieszka Trzewik
+ */
+@Test
+public class CollectionsTest {
+
+    private Book book;
+
+
+    @BeforeMethod
+    public void initialize() {
+        //TODO: Implement these authors and add them as the authors of the book:
+        Author author1;
+        Author author2;
+
+        book = new Book();
+
+    }
+
+    @Test
+    public void givenJavaHashSet_ThenReturnJavaHashSet() {
```

Comment:
Minor and can be ignored: it gave me some pause when I read given - then in test name and had seen AAA in test body. I'd keep AAA and renamed the tests like so: `collectionIsOfTypeHashSet` `persistingHashSetGivesHibernatePersistentSet`.